playground/infrastructure/ci_helper.py

```diff
@@ -0,0 +1,132 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import os
+from typing import List
+
+from api.v1.api_pb2 import Sdk, SDK_JAVA, SDK_GO, SDK_PYTHON, SDK_SCIO
+
+possible_sdk = {'java': SDK_JAVA, 'go': SDK_GO, 'py': SDK_PYTHON, 'scala': SDK_SCIO}
+
+
+def _find_examples(work_dir: str) -> List[str]:
+    """ Find and return filepath to beam examples/tests/katas.
+
+    Search throws all child files of work_dir directory files with beam tag:
+    /*
+    Beam-playground:
+        name: NameOfExample
+        description: Description of NameOfExample.
+        multifile: false
+        categories:
+            - category-1
+            - category-2
+    */
+
+    Args:
+        work_dir: directory where to search examples.
+
+    Returns:
+        List of paths to all tagged files.
+    """
+    examples = [""]
+    return examples
+
+
+def _get_sdk(example) -> Sdk:
+    """ Return SDK of example.
+
+    Get extension of the example by his filepath and returns associated SDK.
+
+    Args:
+        example: filepath of the beam example.
+
+    Returns:
+        Sdk according to file extension.
+    """
+    extension = example.split(os.extsep)[-1]
+    match extension:
+        case 'java':
+            return possible_sdk[extension]
+        case _:
+            raise ValueError(extension + " is not supported now")
```

Comment:
Isn't this match-case from python 3.10? I suppose, we should double-check the version of python in Beam to be sure that we can use such constructions

Reply:
Also, we shouldn't hardcode 'java' this way, let's create a separate variable that will contain all supported SDKs

Reply:
Changed.

Reply:
regarding `raise ValueError(...)`: do you have any suggestions on how I could raise error here?

Reply:
I think you do it right, just a small improvement suggestion:
```python
        extension = example.split(os.extsep)[-1]
        if SUPPORTED_SDK.get(extension):
            return SUPPORTED_SDK[extension]
        else:
            raise ValueError(extension + " is not supported now")
```

Reply:
Changed.